src/app/home/home.page.html

```diff
@@ -1,8 +1,65 @@
 <app-application-header [config]="headerConfig" (actionClick)="handleActionClick($event)"></app-application-header>
 <ion-content [fullscreen]="true">
-  <!-- <app-carousel [items]="carouselItems" [template]="itemTemplate" [itemsToShow]="4"></app-carousel>
+  <div class="ion-margin-vertical">
+    <app-section-header [headerName]="'My Improvements'" [showViewmore]="myImprovements.length > 0" [navigateUrl]="'/my-improvements-listing'"></app-section-header>
+    @if (myImprovements.length > 0) {
+
+      <app-carousel
+        [items]="myImprovements"
+        [template]="itemTemplate1"
+        [itemsToShow]="4"
+      ></app-carousel>
+
+      <ng-template #itemTemplate1 let-item="$item" let-index="$index">
+        <app-myimprovement-card
+        [myImprovement]="item"
+          [index]="index"
+        ></app-myimprovement-card>
+      </ng-template>
+    }
+    @else {
+      <div style="display: flex; justify-content: center;">
+        <mat-card >
+          <img class="card-image" src="assets/MI-2.0 card images/recommendedimg.png" alt="Card image" />
```

Comment:
the image path shouldn't contain any space
Also I don't see this image in this PR

Reply:
removed the blank spaces and image i will be pushing